### PR TITLE
update ADLA version to 0.5.0

### PR DESCRIPTION
**Pull request by lewu-msft**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ x ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [ x ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by troydai** [comment]
Fix CI, please.

**Comment by troydai** [comment]
You make need to re-record these tests.

**Comment by troydai** [comment]
I will add this to ignite release as a potential candidate.

**Comment by troydai** [comment]
Ping @lewu-msft

File: src/command_modules/azure-cli-dla/setup.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
     cmdclass = {}
 
 
-VERSION = "0.2.3"
+VERSION = "0.2.4"
```
Comment (troydai) [inline src/command_modules/azure-cli-dla/setup.py]: No need. `0.2.3` has been released yet.

File: src/command_modules/azure-cli-dla/HISTORY.rst [hunk 2]
```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+0.2.4
+++++++
+* Update azure-mgmt-datalake-analytics dependency to 0.5.0
```
Comment (troydai) [inline src/command_modules/azure-cli-dla/HISTORY.rst]: Combine this with `0.2.3`

**Comment by troydai** [comment]
The tests are still failing. Did you run them locally before pushing the commits?

**Comment by lewu-msft** [comment]
> The tests are still failing. Did you run them locally before pushing the commits?

@troydai I got this error for test_adla_account_mgmt even when I revert all my changes.
msrestazure.azure_exceptions.CloudError: Azure Error: InvalidArgument
Message: ObjectId 8ce05900-7a9e-4895-b3f0-0fbcee507803 of compute policy 'pycliuserpolicy' is invalid. It's weird

**Comment by troydai** [comment]
It is a data lake team's test, after all, I think you're better equipped to fix the test. It is not likely an CLI core issue

**Comment by troydai** [comment]
Ping, what's the progress?

**Pull request closed without merging**